lib/elastic_record/tasks/index.rake

```diff
@@ -32,6 +32,19 @@ namespace :index do
   desc "Recreate index for CLASS or all models."
   task reset: ['index:drop', 'index:create']
 
+  desc "Create index for all models that don't already have one"
+  task create_missing: :environment do
+    ElasticRecord::Task.get_models.each do |model|
+      index = model.elastic_index
+      if (all_names = index.all_names).empty?
+        index_name = index.create_and_deploy
+        puts "Created #{model.name} index (#{index_name})"
+      else
+        puts "#{model.name} already has #{'index'.pluralize(all_names.size)} (#{all_names.to_sentence})"
```

Comment:
This output will be a huge mess for those of us who have had our development environments around for quite some time.  How about just showing the latest index name?  (Or the _deployed_ index, though I couldn't figure out how to get that after 5 minutes of hunting.  It looks like we have an alias that points to the latest index…)

```suggestion
        puts "#{model.name} already has #{'index'.pluralize(all_names.size)} (#{all_names.last})"
```

Reply:
I tested this locally and it only ever prints one index name per model anyway.  But you're right, `.last` is better since it makes it clear what to expect.

Reply:
Fixed and squashed